pkg/commands/secretstoreentry/create.go

```diff
@@ -56,8 +57,20 @@ func NewCreateCommand(parent cmd.Registerer, g *global.Data, m manifest.Data) *C
 	c.RegisterFlag(cmd.StoreIDFlag(&c.Input.ID))  // --store-id
 
 	// Optional.
-	c.RegisterFlag(secretFileFlag(&c.secretFile))       // --file
-	c.RegisterFlagBool(c.JSONFlag())                    // --json
+	c.RegisterFlag(secretFileFlag(&c.secretFile)) // --file
+	c.RegisterFlagBool(c.JSONFlag())              // --json
+	c.RegisterFlagBool(cmd.BoolFlagOpts{
+		Name:        "recreate",
+		Description: "Create or recreate if a secret of the same name already exists within the store",
+		Dst:         &c.recreate,
+		Required:    false,
+	})
+	c.RegisterFlagBool(cmd.BoolFlagOpts{
+		Name:        "recreate-must",
```

Comment:
I'm not really a fan of the naming of these flags because of the overloaded nature of the first one.

To me "recreate" means the thing exists and I want it to be recreated. The fact that it will internally behave like a "create", if _by chance_ I happen to have misspelt the name, is more of an unexpected side-effect I think.

It makes `recreate-must` a bit confusing.

I wonder if we rename `recreate` to `recreate-allowed` and `recreate-must` to `recreate`, like so...

```shell
fastly secrets-store-entry create
# Create secret but error if secret name already exists (i.e. POST) 
# e.g. I think the secret doesn't exist, so I'm OK getting an error back telling me it does

fastly secrets-store-entry create --recreate
# Recreate secret but error if secret name doesn't already exists (i.e. PATCH) 
# e.g. I think the secret already exists, so I explicitly state my expectation is a recreate.

fastly secrets-store-entry create --recreate-allowed
# Create or Recreate secret (i.e. PUT)
# e.g. I don't know if the secret already exists, so I'm saying "allow a replacement to happen"
```

Reply:
I've made your suggested change in https://github.com/fastly/cli/pull/930/commits/263ef6267bf6f6433e6b7482f4850a77d795c32e